x-pack/plugins/cases/server/saved_object_types/cases.ts

```diff
@@ -145,7 +145,7 @@ export const createCaseSavedObjectType = (
         },
       },
       status: {
-        type: 'keyword',
+        type: 'short',
```

Comment:
For the record, we expect 8.7.0 to be the last Kibana version allowing mapping mutations. After that, one would have to take the approach of adding a new field for each mutation. (same comment as was made in https://github.com/elastic/kibana/pull/147498).
Another option is to mutate-on-read if possible.

Reply:
Thank you @TinaHeiligers! We are aware of that. We try to do as much as we can in this release as it is our last opportunity to do so.